### PR TITLE
feat(update-version): update versions on files

### PR DESCRIPTION
**Pull request by alejandrogr**
# Motivation

When creating a tag for new version (`rc`, `fix` or `final`) sometimes we also want to update version files. In this scenario, the tag **must** be created after the version change. 

Given that premise make sense that the action have the capability to change the versions files, commit them and then tag that commit with the new version.

# Technical Details

In order to update the version files we are using [yaml library](https://eemeli.org/yaml/#tostring-options). This restrict the versions files to ymls files (More types of version files can be added in new features)

Four new params was added to the action:

| param | desc |
| --- | --- |
| update-versions-in | String representation of an array with objects specifying files and properties path inside the files to be updated with the new version. eg. `'[{"file": "metaapp/values.yaml", "property": "app.tag"}, {"file": "metaapp/Chart.yaml", "property": "appVersion"}]'` | 
| commit-message | The commit message when commiting the updated version files |
| commit-author | The commit author |
| commit-author-email | The commit author email |

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -29,9 +29,25 @@ inputs:
...
+    required: false
+    default: 'Update version file [skip-ci]'
+  commit-author:
+    description: 'The author for the commit in the version updates.'
+    required: false
+    default: 'Tagger Action'
+  commit-author-email:
+    description: 'The author email for the commit in the version updates.'
+    required: false
+    default: 'tagger-action@automated.com'
```
Comment (guumaster) [inline action.yml]: maybe use an IGZ email, automated.com probably exists.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "main": "index.js",
   "dependencies": {
     "@actions/core": "1.6.0",
-    "@actions/github": "4.0.0"
+    "@actions/exec": "^1.1.0",
```
Comment (guumaster) [inline package.json]:
```suggestion
    "@actions/exec": "1.1.0",
```

File: action.yml (hunk 1 above)
Comment (alejandrogr) [reply action.yml]: fixed in e7fd9c6

File: package.json (hunk 2 above)
Comment (alejandrogr) [reply package.json]: fixed in ff39db1

**Pull request merged**